java/ql/src/Likely Bugs/Likely Typos/SuspiciousDateFormat.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+Some <code>SimpleDateFormat</code> patterns might not work correctly at the end of the calendar
+year, due to use of the <code>Y</code> placeholder (representing the ISO 8601 week year), rather
+than <code>y</code> representing the actual year.
+</p>
+</overview>
+
+<recommendation>
+<p>
+  Ensure the format pattern's use of <code>Y</code> is correct, and if not replace it with <code>y</code>.
+</p>
+</recommendation>
+
+<example>
+<p>
+  The following example uses the date format <code>YYYY-MM-dd</code>.
+  On the 30th of December 2019, this code will output "2020-12-30", rather than the intended "2019-12-30".
```

Comment:
Probably worth being explict about the fix, something like:
"This can be corrected by replacing <code>YYYY-MM-dd</code> with <code>yyyy-MM-dd</code>."